### PR TITLE
Add syntax gram of `load stats`

### PR DESCRIPTION
**Pull request by qw4990**
### What is changed, added or deleted? (Required)

Add syntax gram of `load stats`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by bb7133** [comment]
Please fix the lint.

File: sql-statements/sql-statement-load-stats.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: LOAD STATS
+summary: TiDB 数据库中 LOAD STATS 的使用概况。
+category: reference
+aliases: ['/docs-cn/dev/reference/sql/statements/load-stats/']
```
Comment (zimulala) [inline sql-statements/sql-statement-load-stats.md:5]: 对于新文件不需要 “aliases”。

File: sql-statements/sql-statement-load-stats.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+![LoadStatsStmt](/media/sqlgram/LoadStatsStmt.png)
+
+## 参数说明
+
+用户直接指定统计信息文件路径，统计信息文件可通过访问 API `http://${tidb-server-ip}:${tidb-server-status-port}/stats/dump/${db_name}/${table_name}` 进行下载。
+
+{{< copyable "sql" >}}
+
+```sql
+LOAD STATS '/tmp/stats.json';
```
Comment (zimulala) [inline sql-statements/sql-statement-load-stats.md:25]: 这个只能是绝对路径吗？

Comment (qw4990) [reply sql-statements/sql-statement-load-stats.md:25]: 相对路径也可以其实，需要再补一个例子吗

Comment (zimulala) [reply sql-statements/sql-statement-load-stats.md:24]: 如果相对路径应该是在启动 `tidb-server` 路径下？可以说明一下。

Comment (qw4990) [reply sql-statements/sql-statement-load-stats.md:24]: 行我补充一下

Comment (qw4990) [reply sql-statements/sql-statement-load-stats.md:28]: 更新了 @zimulala

**Comment by yikeke** [comment]
duplicate with https://github.com/pingcap/docs-cn/pull/3362

**Pull request merged**